dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -81,6 +86,179 @@ def test_add_profilers(self):
         self.assertEqual(3, merged_compiler._profiles["test"])
         self.assertEqual("compiler1", merged_compiler.name)
 
+    def test_compiler_stats_diff(self):
```

Comment:
this is not new, I moved the primitive tests below to be int he primitive compiler test class